### PR TITLE
Single-DAG DataStore load flow using CKAN DataStore API

### PR DESCRIPTION
**Pull request by hannelita**
[datapusher-style]; relates to #10

### CODE REVIEW

**Comment by hannelita** [comment]
Note: Opening a separate PR for docs

File: dags/api_ckan_load.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+single_dag = DAG(
+    dag_id='ckan_api_load_single_step',
+    default_args=single_dag_args,
+    schedule_interval=None,
+    tags=['api_load']
+)
+
+def full_load(csv_input, json_output, **kwargs):
```
Comment (rufuspollock) [inline dags/api_ckan_load.py:128]: What is `full_load`? Ah, ok it's a task that does it all in one?

File: dags/api_ckan_load.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    logging.info('Invoking Create Datastore')
+    resource = create_datastore_table(data_resource, get_config())
+    logging.info('Converting resources to json')
+    convert(csv_input, json_output)
+    logging.info('Loading CSV via API')
+    try:
+        with open(json_output) as f:
+            records = json.load(f)
+            return load_csv_via_api(
```
Comment (rufuspollock) [inline dags/api_ckan_load.py:137]: `load_csv_via_api` should provbably be renamed: `load_records_via_api`

File: lib/load.py [hunk 3]
```diff
@@ -65,14 +66,73 @@ def create_datastore_table(data_resource, config={}):
...
-            return {'success': True, 'response_resource_id': resource_json}
+            return {'success': True, 'response_resource': resource_json, 'response_resource_id': resource_id}
             log.info('Table was created successfuly')
         else:
             return response.json()
     except Exception as e:
         return {"success": False, "errors": [e]}
 
 
+# def send_resource_to_datastore(resource, headers, records,
```
Comment (rufuspollock) [inline lib/load.py:77]: Are we going to uncomment at some point? Otherwise feel free to drop.

**Pull request merged**